types/index.d.ts

```diff
@@ -0,0 +1,565 @@
+
+  /**
+   * Return an object with several statistics. Return value looks similar to
+   * (but can be subject to change in future versions or different
+   * configurations, so use defensively):
+   *
+   * ```javascript
+   * {
+   *     "cpu": {
+   *         "instruction_counter": 2821610069
+   *     },
+   *     "hda": {
+   *         "sectors_read": 95240,
+   *         "sectors_written": 952,
+   *         "bytes_read": 48762880,
+   *         "bytes_written": 487424,
+   *         "loading": false
+   *     },
+   *     "cdrom": {
+   *         "sectors_read": 0,
+   *         "sectors_written": 0,
+   *         "bytes_read": 0,
+   *         "bytes_written": 0,
+   *         "loading": false
+   *     },
+   *     "mouse": {
+   *         "enabled": true
+   *     },
+   *     "vga": {
+   *         "is_graphical": true,
+   *         "res_x": 800,
+   *         "res_y": 600,
+   *         "bpp": 32
+   *     }
+   * }
+   * ```
+   *
+   * @deprecated
+   */
+  get_statistics(): Object
+  get_instruction_counter(): number
+  is_running(): boolean
+
+  /**
+   * Send a sequence of scan codes to the emulated PS2 controller. A list of
+   * codes can be found at http://stanislavs.org/helppc/make_codes.html.
+   * Do nothing if there is no keyboard controller.
+   *
+   * @param codes
+   */
+  keyboard_send_scancodes(codes: number[]): void
+  
+  /**
+   * Send translated keys
+   */
+  keyboard_send_keys(codes: any[]): void
+
+  /**
+   * Send text
+   */
+  keyboard_send_text(string: string): void
+
+  
+  /**
+   * Download a screenshot.
+   */
+  screen_make_screenshot(): void
+
+  /**
+   * Set the scaling level of the emulated screen.
+   *
+   * @param {number} sx
+   * @param {number} sy
+   *
+   * @ignore
+   * @export
+   */
+  screen_set_scale(sx: number, sy: number): void
+
+  /**
+   * Go fullscreen.
+   */
+  screen_go_fullscreen(): void
+
+  /**
+   * Lock the mouse cursor: It becomes invisble and is not moved out of the browser window.
+   */
+  lock_mouse(): void
+
+  /**
+   * Enable or disable sending mouse events to the emulated PS2 controller.
+   */
+  mouse_set_status(enabled: boolean): void
+  
+  /**
+   * Enable or disable sending keyboard events to the emulated PS2 controller.
+   */
+  keyboard_set_status(enabled: boolean): void
+
+  /**
+   * Send a string to the first emulated serial terminal.
+   *
+   * @param data
+   */
+  serial0_send(data: string): void
+
+  /**
+   * Send bytes to a serial port (to be received by the emulated PC).
+   *
+   * @param serial the index of the serial port
+   * @param data
+   */
+  serial_send_bytes(serial: number, data: Uint8Array)
+
+  /**
+   * Mount another filesystem to the current filesystem.
+   * @param path Path for the mount point
+   * @param baseurl
+   * @param basefs As a JSON string
+   * @param callback
+   * @export
+   */
+  mount_fs(path: string, baseurl?: string, basefs?: string, callback?: (error: Object | null) => void): void
```

Comment:
Could you remove this? (optionally also comment it out in the souce file), I don't think it's functional).